### PR TITLE
[OneExplorer] Use extensionKind to know whether extensions' remote

### PR DESCRIPTION
**Pull request by dayo09**
This commit uses extensionKind instead of remoteName
to know if the extension is remote or not.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

----

For #1209
/cc @jyoungyun @hyunsik-yoon

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
Looks good to me :) Thank you.

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -621,7 +648,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     let approval: string;
     let useTrash: boolean;
 
-    if (vscode.env.remoteName) {
+    if (this.isRemote) {
```
Comment (llFreetimell) [inline src/OneExplorer/OneExplorer.ts:651]: Maybe...?

```suggestion
    if (this.isRemote()) {
```

Comment (dayo09) [reply src/OneExplorer/OneExplorer.ts:651]: It's a getter, so it doesn't need parenthesis :-D

https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/

**Comment by hyunsik-yoon** [comment]
I checked this in my SSH-extension env and also in a single Linux box env.
In both case, `_extensionKind` has value described by @dayo09.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**